CRM/Twingle/Profile.php

```diff
@@ -66,6 +66,9 @@ public function __construct($name, $data) {
   public function matches($project_id) {
     $selector = $this->getAttribute('selector');
     $project_ids = explode(',', $selector);
+    for ($i=0; $i<sizeof($project_ids); $i++) {
+      $project_ids[$i] = trim($project_ids[$i]);
+    }
```

Comment:
How about using an array utility function instead?

```php
$project_ids = array_map(
  function($project_id) {
    return trim($project_id);
  },
  explode(',', $selector)
);
```

Reply:
Yes! This is definitely much more elegant!

Reply:
Okay, I tested it. This solution works as well.